prior:
  base_orientation: [1,0,0,0]
  base_velocity: [0,0,0]
  base_position: [0,0,0]
  gyroscope_bias: [0,0,0]
  accelerometer_bias: [0,0,0]
  base_velocity_std: 0.1
  base_orientation_std: 0.001
  base_position_std: 0.01
  gyroscope_bias_std: 0.001
  accelerometer_bias_std: 0.01
  landmarks: 
    - id: 0
      position: [1.0,2.0,3.0] 
    - id: 1
      position: [4.0,5.0,6.0]
    - id: 2
      position: [7.0,8.0,9.0]